splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/index/IndexTransformer.java

```diff
@@ -223,9 +223,8 @@ public KVPair encodeSystemTableIndex(ExecRow execRow) throws StandardException,
         byte[] key = DerbyBytesUtil.generateIndexKey(dvds, order,tableVersion,false);
 
         if(entryEncoder==null){
-            int[] validCols= EngineUtils.bitSetToMap(null);
             DescriptorSerializer[] serializers=VersionedSerializers.forVersion(tableVersion,true).getSerializers(execRow);
-            entryEncoder=new EntryDataHash(validCols,null,serializers);
```

Comment:
Oh it is not needed, it is a small refactoring, because if you pass `null` to `EngineUtils.bitSetToMap` you get back `null`. I thought I make the intended usage clear here by directly passing the `null`. I can revert it if you think it irrelevant.

Reply:
That make sense, I also think it's clearer this way, thanks!